src/Uno.UI.RemoteControl/HotReload/ClientHotReloadProcessor.Xaml.cs

```diff
@@ -109,7 +109,7 @@ bool IsSameBaseUri(FrameworkElement i)
 							break;
 #endif
 						case ContentControl content:
-							if (XamlReader.LoadUsingXClass(fileContent) is ContentControl newContent)
+							if (XamlReader.LoadUsingXClass(fileContent, uri.ToString()) is ContentControl newContent)
```

Comment:
Not deeply familiar with how remote control works, but we need to make sure whether `XamlReader.Load` call few lines before need similar change.

Note: `XamlReader.Load` is public, so if we need to pass it the uri we will add a new internal overload.

Reply:
This doesn't use `XamlReader.Load` it uses `XamlReader.LoadUsingXClass` which is already internal.
I will change the use of `XamlObjectBuilder` so that the public ctor is unchanged and add a suitable internal variation there instead.

Reply:
> This doesn't use `XamlReader.Load` it uses `XamlReader.LoadUsingXClass` which is already internal.

Yes, I was referring to the un-updated `Load` call. If we will update it, then another overload is needed.